apps/docs/content/components/internal-parts/PopoverBase.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: PopoverBase
+description: A simple React Aria Popover with hopper's styling
+category: "internal parts"
+links:
+    source: https://github.com/workleap/wl-hopper/blob/main/packages/components/src/overlays/popover/src/PopoverBase.tsx
+---
+
+The PopoverBase component is an internal building block that provides the core popover functionality with Hopper's styling.
```

Comment:
1- Le text en haut + when to use est trop similaire.
2- Je pense renommé les internal parts à genre building blocks ou quelque chose du genre dans le futur. Dans le texte on pourrait ne pas mettre le mot internal.

Je pense qu'on peux remplacer les 2 premier paragraph par quelque chose du genre: 

> PopoverBase is a low-level building block that wraps React Aria's Popover to provide core popover functionality with Hopper's design system styling and enhancements. It doesn't impose layout or padding, making it ideal for building custom popover structures from scratch.